### ISSUE TITLE
Stalls uploading on big .flacs

### ISSUE THREAD

**Issue by kidcodea** [jamesbrindle/YTMusicUploader#11]
Works fine, but uploading stalls when it meets a big .flac in your HD like entire live shows in one .flac type of files. Usually in the >400MB region. Unfortunately can't be skipped either manually or (ideally) automatically.

**Comment by jamesbrindle** [comment]
Hi, the max file size is 300MB... I have accounted for this in code I believe, but honestly, I've not done much checking on that as I haven't a 400MB file to check. If you put a file for me to test with here: https://1drv.ms/u/s!At6EvCXZJAN6zZ9BuFsgs9qslLMZqA?e=KQ4I8G  I'll make sure those kind of files are skipped. With a message in the log (and status bar at the time of skipping)

**Comment by kidcodea** [comment]
No idea why impose such limit since so many files go past that and prevent a successful library sync to ytmusic, but since you set it at that i'm gonna presume you had some reason to. Of course ideally such a distinction wouldn't exist and files of all sizes ytmusic can accept could be uploaded, but if you could at least make these files auto-magically-skip, that would be a big boost in usability already. I have put a 458MB file in the place you asked for, so u can see how currently such a file stalls the entire process and it doesn't advance in upload queue. Good luck and thank you for your efforts.

**Comment by captiglo** [comment]
Just want to add that I have exactly the same problem. I also observed that YTMusicuploader.exe puts 100% CPU load on my system when it reaches the FLAC file. Status says "Gathering uploaded artists from YouTube Music" and this does not change over hours.

**Comment by jamesbrindle** [comment]
Just to let you know... I was supposed to look at fixing this last weekend, but ended up working all weekend. I've made a good number of changes though tonight, which could address the issue. If not... I've introduced some decent logging, so if the problem persists I'm hopeful I can get to the bottom of it. I haven't deployed the changes yet... I've got a couple more things to do and I'll get this pushed tomorrow afternoon.
